src/app/components/utility/side-bar/side-bar.component.scss

```diff
@@ -1,13 +1,11 @@
 @import "styles/variables";
 .ev-sidebar {
   display: block;
-  height: calc(100vh - 120px);
+  height: 100vh;
   width: 220px;
-  border-right: 1px solid $grad-light-black;
-  background-color: $transparent;
+  background-color: #fafafa;
```

Comment:
Now, we don't this change here.

Reply:
we still need it because when we scroll to the bottom the sidebar will be messed up

Reply:
because of the footer

Reply:
but why?

Reply:
because the side bar is fixed

Reply:
if the sidebar doesnt have that background color when we scroll over the footer it will not show properly. You cann test it and see

Reply:
Yes, I've tested, it's working on my local.